aten/src/ATen/function_wrapper.py

```diff
@@ -1158,6 +1173,9 @@ def add_namedtensor_enabled_macro(code):
                 check_namedtensor_enabled(NATIVE_DISPATCH_DEFINITION_DEFAULT.substitute(option)))
             top_env['function_registrations'].append(
                 check_namedtensor_enabled(DEFAULT_FUNCTION_REGISTRATION.substitute(option)))
+            if not option['exclude_from_c10_dispatcher']:
```

Comment:
My main concern with this is that if we have different registrations, it will make backend extensions very confusing. I guess this patch itself seems fine, but once if we start calling through c10 path, it will be confusing.

Reply:
Do you think we can have globalATenDispatch forward registration calls to c10 as a temporary solution? This way, we only have to switch backends once when all ops are on c10.

Reply:
Yeah I think that's reasonable, especially if we're already generating a list of exclusions for the jit registry, we can just use it for this purpose as well.